composer.json

```diff
@@ -20,8 +23,23 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "BEdita\\AWS\\Test\\": "tests",
-            "Cake\\Test\\": "./vendor/cakephp/cakephp/tests"
+            "BEdita\\AWS\\Test\\": "tests"
+        }
+    },
+    "scripts": {
+        "cs-setup": [
+            "vendor/bin/phpcs --config-set installed_paths vendor/cakephp/cakephp-codesniffer",
```

Comment:
Instead of `cs-setup` I suggest to use `phpcs.xml` file, for example
```xml
<?xml version="1.0"?>
<ruleset name="App">
    <arg name="colors"/>
    <config name="installed_paths" value="../../cakephp/cakephp-codesniffer"/>

    <rule ref="CakePHP"/>
</ruleset>
```

Reply:
Totally agree! Thanks!